subseas_flex_fcst/maproom.py

```diff
@@ -39,6 +46,53 @@
 
 APP.layout = layout.app_layout
 
+
+def adm_borders(shapes):
+    with psycopg2.connect(**CONFIG["db"]) as conn:
+        s = sql.Composed(
+            [
+                sql.SQL("with g as ("),
+                sql.SQL(shapes),
+                sql.SQL(
+                    """
+                    )
+                    select
+                        g.label, g.key, g.the_geom
+                    from g
+                    """
+                ),
+            ]
+        )
+        df = pd.read_sql(s, conn)
+
+    df["the_geom"] = df["the_geom"].apply(lambda x: wkb.loads(x.tobytes()))
+    df["the_geom"] = df["the_geom"].apply(
+        lambda x: x if isinstance(x, MultiPolygon) else MultiPolygon([x])
+    )
+    shapes = df["the_geom"].apply(shapely.geometry.mapping)
+    for i in df.index: #this adds the district layer as a label in the dict
+        shapes[i]['label'] = df['label'][i]
+    return {"features": shapes}
+
+
+def make_adm_overlay(adm_name, adm_sql, adm_color, adm_lev, adm_weight, is_checked=False):
+    border_id = {"type": "borders_adm", "index": adm_lev}
```

Comment:
Why did you use a compound id instead of a string? The only reason I know of is for use with a pattern-matching callback, but I don't see any of those in this PR.

I think you can just not assign this component an id at all. That would let you eliminate the `adm_lev` parameter.

Reply:
That's because it's coming form the onset maproom set up that is using this to print the name of the finest admin level label when mousing hover. So it will be needed or used when that part is implemented. It's just not there yet. That's likely a feature we'd want very often if not standard, so shall we keep this here when the mouse hover is put up?

Reply:
ok